app.rb

```diff
@@ -4,21 +4,29 @@
 require_relative './teacher'
 require_relative './book'
 require_relative './classroom'
+require_relative './load_data'
+require 'json'
 
 class App
   puts "Welcome to school library app!\n\n"
   def initialize
     @books = []
     @people = []
     @rentals = []
+    loader = Loader.new
+    loader.load_people(@people)
+    loader.load_books(@books)
+    loader.load_rentals(@rentals)
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly your `Exit` option has a error therefore I can't find `loaded` files, please fix it🙏🏼

![image](https://user-images.githubusercontent.com/63915024/232797437-43335014-ad8d-40a1-ad2b-4bef96811d5f.png)

Reply:
Aye Aye Captain 👏